DFC.ServiceTaxonomy.ApiFunction/DFC.ServiceTaxonomy.ApiFunction.csproj

```diff
@@ -15,4 +26,15 @@
       <CopyToPublishDirectory>Never</CopyToPublishDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <CypherQueryFiles Include="$(ProjectDir)\CypherQueries\*.json" />
+  </ItemGroup>
+  
+  <Target Name="CopyCustomContentOnPublish" AfterTargets="Publish" Condition="'$(Configuration)' == 'Release'">    
```

Comment:
If you're going to do anything else on this branch,  I'd remove this condition..  it's currently stopping this working once deployed, as we're currently building in debug mode..